test/test-ghe-backup.sh

```diff
@@ -214,6 +213,86 @@ begin_test "ghe-backup subsequent snapshot"
 )
 end_test
 
+begin_test "ghe-backup with relative data dir path"
+(
+    set -e
+
+    # wait a second for snapshot timestamp
+    sleep 1
+
+    # generate a timestamp
+    export GHE_SNAPSHOT_TIMESTAMP="relative-$(date +"%Y%m%dT%H%M%S")"
+
+    # change working directory to the root directory
+    cd $ROOTDIR
+
+    # run it
+    GHE_DATA_DIR=$(echo $GHE_DATA_DIR | sed 's|'$ROOTDIR'/||') ghe-backup
```

Comment:
My reasoning for only passing the relative path through to `ghe-backup` and not the rest of the test is so that the full absolute path is used for the tests.